Classes/DataHandler/AbstractDataHandler.php

```diff
@@ -26,7 +26,7 @@
 use Doctrine\DBAL\Driver\Exception as DBALDriverException;
 use Doctrine\DBAL\Exception as DBALException;
 use EHAERER\PasteReference\Helper\Helper;
-use TYPO3\CMS\Core\Database\Connection;
+use TYPO3\CMS\Core\Database\Connection as Typo3Connection;
```

Comment:
Is there a reason to set is as `Typo3Connection`? There is no other `Connection` inside this class.

Reply:
mhm, artists inspiration could serve as only argument